integration-cli/docker_cli_commit_test.go

```diff
@@ -116,9 +116,9 @@ func (s *DockerSuite) TestCommitChange(c *check.C) {
 		"--change", "ENV test 1",
 		"--change", "ENV PATH /foo",
 		"--change", "LABEL foo bar",
-		"--change", "CMD [\"/bin/sh\"]",
+		"--change", `"CMD [""/bin/sh""]"`,
```

Comment:
@dnephin I tried with `ListOpts` and it woks as expected. Thanks for the help!

The pull request has been updated. Please let me know if there are any issues.